src/Entities/Tetrahedrons.jl

```diff
@@ -89,57 +96,55 @@ function _jacobian_mat(tetrahedron_coords_matrix::AbstractMatrix, derivatives::A
     tetrahedron_coords_matrix * derivatives'
 end
 
-"Computes volume element of a tetrahedron given J = det(𝔽)."
+"Computes volume element of a tetrahedron given J = det(F)."
 function _volume(jacobian_mat::AbstractMatrix)
     volume = det(jacobian_mat) / 6.0
     @assert volume > 0 throw(ArgumentError("Element with negative volume, check connectivity."))
     volume
 end
 
-function _B_mat(deriv::AbstractMatrix, 𝔽::AbstractMatrix)
+function _B_mat(deriv::AbstractMatrix, F::AbstractMatrix)
```

Comment:
I think you can also pass  `B` first arg to `_B_mat!` and cache it.

Reply:
Done, I have also passed aux which is a 4x4 matrix

Reply:
Done